falcon/response.py

```diff
@@ -713,24 +720,34 @@ def add_link(self, target, rel, title=None, title_star=None,
 
         """)
 
-    def _wsgi_headers(self, media_type=None, py2=PY2):
-        """Convert headers into the format expected by WSGI servers.
+    def _set_media_type(self, media_type=None):
+        """wrapper around set_header to set a content-type
```

Comment:
[nitpick] Capitalize "wrapper" and end the first line of the doctoring with a period.